WordPress/Classes/Utility/WPImmuTableRows.swift

```diff
@@ -236,6 +236,34 @@ struct LinkWithValueRow: ImmuTableRow {
     }
 }
 
+/// Create a row button: uses the WordPress branded blue and is left aligned
+///
+struct BrandedButtonRow: ImmuTableRow {
```

Comment:
Maybe we can give this type a name that clarifies a bit better what it is. The cell does not actually contain a button, and it's specifically used in App Settings, so Maybe we can use something like `SettingsActionRow` or similar? What do you think?